packages/backend-test-utils/src/next/implementations/mockIdentityService.ts

```diff
@@ -13,17 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import {
-  coreServices,
-  createServiceFactory,
-  IdentityService,
-} from '@backstage/backend-plugin-api';
+import { IdentityService } from '@backstage/backend-plugin-api';
 import {
   IdentityApiGetIdentityRequest,
   BackstageIdentityResponse,
 } from '@backstage/plugin-auth-node';
 
-class MockIdentityServiceImpl implements IdentityService {
+export class MockIdentityService implements IdentityService {
```

Comment:
i guess now this file would be best named like the class (with an uppercase M), likewise for any others that now contain just the class

Reply:
Yep, for this one I like to avoid case-only changes though, can mess up things on some filesystems

Reply:
I ended up renaming the dir to `services`, so could rename all of the implementations too